### PR TITLE
add /beacon/validators post taking pubkey list and epoch

### PR DESCRIPTION
**Pull request by rolfyone**
addresses #1203

### CODE REVIEW

File: config/config.toml [hunk 1]
```diff
@@ -51,4 +51,4 @@ startFromDisk = false
 
 [beaconrestapi]
 portNumber = 5051
-enableSwagger = false
+enableSwagger = true
```
Comment (rolfyone) [inline config/config.toml]: ok thats not good.

File: data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/PostValidators.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import tech.pegasys.artemis.api.ChainDataProvider;
+import tech.pegasys.artemis.api.schema.BeaconValidators;
+import tech.pegasys.artemis.api.schema.ValidatorWithIndex;
+import tech.pegasys.artemis.api.schema.ValidatorsRequest;
+import tech.pegasys.artemis.beaconrestapi.schema.BadRequest;
+import tech.pegasys.artemis.provider.JsonProvider;
+
+public class PostValidators implements Handler {
+
+  @SuppressWarnings("unused")
```
Comment (mark-terry) [inline data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/PostValidators.java]: Is this still showing as unused?

File: data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/PostValidators.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  public static final String ROUTE = "/beacon/validators";
+  private final JsonProvider jsonProvider;
+
+  @OpenApi(
+      path = ROUTE,
+      method = HttpMethod.POST,
+      summary = "Returns validators that match the information specified.",
+      tags = {TAG_BEACON},
+      description =
+          "Takes a list of validator public keys and optionally an epoch, and returns information about them.",
```
Comment (mark-terry) [inline data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/PostValidators.java]: Do we need to describe the default behaviour if no epoch is specified?

**Review by mark-terry — APPROVED** [review]
LGTM - a few queries.

File: data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/PostValidators.java (hunk 3 above)
Comment (rolfyone) [reply data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/PostValidators.java]: I'll expand the description to include information about the current state when epoch is missing, and i can put in a line about what happens when pubkeys aren't found too.

File: data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/PostValidators.java (hunk 2 above)
Comment (rolfyone) [reply data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/PostValidators.java]: ooh i hope not! I'll remove that suppression.

**Pull request merged**